app/components/chat.tsx

```diff
@@ -523,7 +544,11 @@ export function ChatActions(props: {
       {showBox && <BoxLogin showModal={showBox} setShowModal={setShowBox} />}
 
       {isUser === "true" ? (
-        <ChatAction text="logout" style="borderBlue" onClick={() => LogOut()} />
+        <ChatAction
+          text="Sign Out"
```

Comment:
why do you need to change the text of the button? This change doesn't align the button to the right?